Examples/FullStackSwiftExample/iOSApp/README.md

```diff
@@ -0,0 +1,16 @@
+# Kittens iOS Frontend
```

Comment:
I think it could be cool to include some screenshots here and maybe in the top level readme too

Reply:
good idea, added screenshots to both